build-image-src/Dockerfile-java17

```diff
@@ -57,6 +57,10 @@ ENV PATH="/usr/local/gradle/gradle-7.3.1/bin:/usr/local/maven/apache-maven-3.8.8
 
 COPY ATTRIBUTION.txt /
 
+# Set following GRADLE_OPTS and MAVEN_OPTS to fix issues with arm64 image
+ENV GRADLE_OPTS="-Djdk.lang.Process.launchMechanism=vfork"
+ENV MAVEN_OPTS="-Djdk.lang.Process.launchMechanism=vfork"
```

Comment:
do we need the customers to set these variables for using `sam build` without container on arm64 environments ?